docs/computer/Acoustics.md

```diff
@@ -63,21 +63,21 @@ Key features:
 3.Inbuilt IMU that can be used further for movement of the vehicle.
 4.WiFi connectivity.
 
-![NI myRIO-1900 Hardware Block Diagram](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRIO_hardware_pinout.png)
+![NI myRIO-1900 Hardware Block Diagram](static/myRIO_hardware_pinout.png)
 
-![Primary/Secondary Signals on MXP Connectors A and B](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRio_ports.png)
-_Source: [NI MyRIO User Guide and Specification](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/NI_MyRIO_User_Guide_and_Specification.pdf)_
+![Primary/Secondary Signals on MXP Connectors A and B](static/myRio_ports.png)
+_Source: [NI MyRIO User Guide and Specification](static/NI_MyRIO_User_Guide_and_Specification.pdf)_
 
 ## The process
 
 ### Placement of hydrophones  
 
 They are placed at specific distances in an equilateral shape for accurate results.
 Reference: [Design and analysis of air acoustic vector-sensor configurations for two-dimensional
-geometry](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/wajid2016.pdf)
+geometry](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/wajid2016.pdf)
 The approximate distance of the placement of the arrays is calculated to be 3 inches apart. The distance is measured from the center of the hydrophones. It has to be in the form of an equilateral triangle, as seen in the figure.
 
-![Star configuration, having four measured signals, i.e., _pc_, _pk_ , _pl_ , and _pm_.](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/microphone_placement.png)
+![Star configuration, having four measured signals, i.e., _pc_, _pk_ , _pl_ , and _pm_.](static/microphone_placement.png)
```

Comment:
Is the written part important to show up? If yes, then you got to write it down again after image.

Reply:
I'll add captions to it and remove the written part.

Reply:
Sounds Good!
Send another patch for this.